src/L2GovernorMetadata.sol

```diff
@@ -21,11 +21,13 @@ contract L2GovernorMetadata is WormholeReceiver {
 
   /// @notice Receives a message from L1 and saves the proposal metadata.
   /// @param payload The payload that was sent to in the delivery request.
-  function receiveWormholeMessages(bytes memory payload, bytes[] memory, bytes32, uint16, bytes32)
-    public
-    override
-    onlyRelayer
-  {
+  function receiveWormholeMessages(
+    bytes memory payload,
+    bytes[] memory,
+    bytes32 sourceAddress,
+    uint16 sourceChain,
```

Comment:
Because the receiver does not need to specify a source chain. The source chain is specified by the relayer. A sender will need to specify a source chain a good example is [here](https://github.com/ScopeLift/l2-flexible-voting/blob/0008de9f707e2f87c892e25953d61d46117d126b/src/L1GovernorMetadataBridge.sol#L63)

Reply:
Ref https://github.com/ScopeLift/l2-flexible-voting/pull/37#discussion_r1323310122 around docs/natspec